docs/extensions/compose.md

```diff
@@ -59,6 +59,37 @@ fun SomeContent(component: SomeComponent) {
 }
 ```
 
+### Lifecycle
+
+When using JetBrains Compose, you can have a `LifecycleRegistry` react to changes in the window state using the `LifecycleController()` composable. This will trigger appropriate lifecycle events when the window is minimized, restored or closed.
+
+It is also possible to manually start the lifecycle using `LifecycleRegistry.resume()` when the instance is created.
+
+```kotlin
+fun main() {
+    val lifecycle = LifecycleRegistry()
+    val root = RootComponent(DefaultComponentContext(lifecycle))
+    
+    // Alternative: manually start the lifecycle (no reaction to window state)
+    // lifecycle.resume()
+    
+    application {
+        val windowState = rememberWindowState()
+        
+        // Bind the registry to the life cycle of the window
+        LifecycleController(lifecycle, windowState)
+        
+        Window(state = windowState, ...) {
+            // The rest of your content
+        }
+    }
+}
+```
+
+
+
+> Warning: When using Compose in desktop platforms, make sure to always use one of the methods above, or your components might not receive lifecycle events correctly.
```

Comment:
There is an established way in the repository to display warnings. Would it be possible for you to change for consistency? See an [example](https://github.com/arkivanov/Decompose/blob/986a60799aa7d8c06e6a4f08f30cb494be646d3f/docs/router/overview.md?plain=1#L34).

```
> ⚠️ Abc
```